source/io/module_eggShell.js

```diff
@@ -405,6 +397,8 @@ var assignEggShell;
                 subTypeRef = Module.typeHelpers.subElementByIndex(typeRef, 0);
                 if (Module.typeHelpers.isBoolean(subTypeRef)) {
                     return Uint8Array;
+                } else if (Module.typeHelpers.isJSObjectRefnum(subTypeRef)) {
```

Comment:
Added to allow bulk reading of JavaScript Refnum cookies from memory. Used by the paramter and returnValueVisitor